flavors/squid_nlb_central/updatewhitelist.sh

```diff
@@ -4,7 +4,7 @@ cd /home/ubuntu/cloud-automation
 git pull
 
 echo "Checking if list of authorized keys have changed for admin"
```

Comment:
Does this script run as the `ubuntu` user in a crontab?  If so - please add a comment explaining that.

Reply:
the cronjob is run as root; so removed sudo